import-export-cli/cmd/importAPI.go

```diff
@@ -158,11 +158,37 @@ func mergeAPI(apiDirectory string, environmentParams *params.Environment) error
 		return err
 	}
 
+	if !isEndpointsFieldsValid(environmentParams.Endpoints, environmentParams.LoadBalanceEndpoints, environmentParams.FailoverEndpoints) {
+		return errors.New("Please specify only one field from endpoints, loadBalanceEndpoints or failOverEndpoints in the api_params.yaml file for " +
```

Comment:
seems like maximum length is exceeded for this line.

Reply:
Fixed via https://github.com/wso2/product-apim-tooling/pull/596/commits/89238ea671a041105f709a8d49c1a3c6732c4e64